tools/build_chain.sh

```diff
@@ -1659,8 +1668,8 @@ for line in ${ip_array[*]};do
             p2p_port=$(grep listen_port < "${node_dir}"/config.ini | grep -v _listen_port | cut -d = -f 2)
             mv "${node_dir}" "${output_dir}/${ip}/node_${ip}_${p2p_port}"
         fi
-        set_value ${ip//./}_count $(( $(get_value ${ip//./}_count) + 1 ))
-        set_value "${ip//./}_port_offset" $(( $(get_value "${ip//./}_port_offset") + 1 ))
+        set_value "${ip//[\.:]/_}_count" $(( $(get_value ${ip//[\.:]/_}_count) + 1 ))
```

Comment:
Codacy found an issue: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/gh/FISCO-BCOS/FISCO-BCOS/pullRequest?prid=5837713)